peer/network.go

```diff
@@ -335,8 +339,8 @@ func (n *network) AppRequest(ctx context.Context, nodeID ids.NodeID, requestID u
 
 	var req message.Request
 	if _, err := n.codec.Unmarshal(request, &req); err != nil {
-		log.Debug("failed to unmarshal app request", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request), "err", err)
-		return nil
+		log.Debug("forwarding unknown AppRequest", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request), "err", err)
```

Comment:
Can we change all of these debug logs to be more specific to what's happening? `unknown request` sounds like an error message, but this is expected behavior during the transition period.

Can we change to `forwarding request unknown to codec to SDK handler` or similar?

Reply:
Patrick disliked the `... to sdk` wording from the first revision so I'll use `forwarding unregistered AppRequest`